components/automate-ui/src/app/modules/infra-proxy/policy-groups-list/policy-groups-list.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-policy-groups-list',
+  templateUrl: './policy-groups-list.component.html'
+})
+export class PolicyGroupsListComponent implements OnInit {
+  @Input() policyGroups: [];
+  public policyGroupsList = [];
```

Comment:
let the name we keep as policyGroups no need to add list add the end

Reply:
It can conflict with decorator name.